versions.yaml

```diff
@@ -205,6 +205,7 @@ externals:
     version: "18c8ee378c6d83446ee635a702d5dee389028d8f"
     image: "ghcr.io/confidential-containers/staged-images/kbs"
     image_tag: "18c8ee378c6d83446ee635a702d5dee389028d8f"
+    rust: "1.74.0"
```

Comment:
nit: you could call this "toolchain" to be more consistent with the rest of versions.yaml